src/components/TextEditor.vue

```diff
@@ -157,10 +162,12 @@ export default {
 
   mounted() {
     EventBus.$on('caption_changed', this.onUpdate);
+    //EventBus.$on('caption_content_update', this.updateContent);
```

Comment:
should this be commented out still?

Reply:
Yeah that was left in accidentally, removed all of it.